### PR TITLE
doc/cephfs: edit "Layout Fields" text

### PR DESCRIPTION
**Pull request by zdover23**
Improve the English in the "Layout Fields" section of doc/cephfs/file-layouts.rst.

## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "quincy"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

- When filling out the below checklist, you may click boxes directly in the GitHub web UI.  When entering or editing the entire PR message in the GitHub web UI editor, you may also select a checklist item by adding an `x` between the brackets: `[x]`.  Spaces and capitalization matter when checking off items this way.

## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [x] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [x] No impact that needs to be tracked
- Documentation (select at least one)
  - [x] Updates relevant documentation
  - [ ] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [x] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
- `jenkins test rook e2e`
</details>

### CODE REVIEW

File: doc/cephfs/file-layouts.rst [hunk 1]
```diff
@@ -20,26 +20,38 @@ Layout fields
 -------------
 
 pool
-    String, giving ID or name. String can only have characters in the set [a-zA-Z0-9\_-.]. Which RADOS pool a file's data objects will be stored in.
+    This is a string and returns either an ID or a name. Strings may contain
+    only characters in the set ``[a-zA-Z0-9\_-.]``. The pool determines the
```
Comment (anthonyeleven) [inline doc/cephfs/file-layouts.rst:24]: s/The pool/This/

File: doc/cephfs/file-layouts.rst [hunk 2]
```diff
@@ -20,26 +20,38 @@ Layout fields
...
-    be written to.  Empty by default (i.e. default namespace).
+    This is a string containing only characters in the set ``[a-zA-Z0-9\_-.]``.
+    This determines which RADOS namespace within the data pool that the objects
+    will be written to.
+    Empty by default (i.e. default namespace).
 
 stripe_unit
-    Integer in bytes.  The size (in bytes) of a block of data used in the RAID 0 distribution of a file. All stripe units for a file have equal size. The last stripe unit is typically incomplete–i.e. it represents the data at the end of the file as well as unused “space” beyond it up to the end of the fixed stripe unit size.
+    This is an integer. The size (in bytes) of a block of data used in the RAID
+    0 distribution of a file. All stripe units for a file have equal size. The
```
Comment (anthonyeleven) [inline doc/cephfs/file-layouts.rst:43]: s/RAID 0//

**Review by anthonyeleven — APPROVED** [review]

File: doc/cephfs/file-layouts.rst (hunk 1 above)
Comment (zdover23) [reply doc/cephfs/file-layouts.rst:24]: Accepted.

File: doc/cephfs/file-layouts.rst (hunk 2 above)
Comment (zdover23) [reply doc/cephfs/file-layouts.rst]: Accepted.

**Comment by zdover23** [comment]
jenkins test docs

**Comment by zdover23** [comment]
https://github.com/ceph/ceph/pull/59021 - Squid backport
https://github.com/ceph/ceph/pull/59022 - Reef backport
https://github.com/ceph/ceph/pull/59023 - Quincy backport

**Pull request merged**